config/default/kustomization.yaml

```diff
@@ -28,7 +28,7 @@ patchesStrategicMerge:
 images:
 - name: kusk-gateway
   newName: kubeshop/kusk-gateway
-  newTag: v1.2.5
+  newTag: v1.3.0
```

Comment:
wouldn't this be v1.3.1?

Reply:
no image 1.3.1. Once we cut the release it will get bumped to v1.3.1